src/main/java/com/apextrio/peak/appuser/AppUser.java

```diff
@@ -0,0 +1,113 @@
+package com.apextrio.peak.appuser;
+
+import com.apextrio.peak.team.Team;
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import java.util.Collection;
+import java.util.Set;
+
+//Class model structure for users.
+@Entity
+public class AppUser implements UserDetails {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    private String username;
+    @JsonIgnore
+    private String password;
+    private String bio;
```

Comment:
When you have some fields with annotations and some without, it's customary to leave newlines between each instance variable you declare; then it's clearer which lines are annotations, which variables they're for, and how many variables you've created.